.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,66 @@
+name: test-windows
+on:
+  pull_request:
+    paths-ignore:
+      - "**.md"
+      - "**.jpg"
+      - "**.png"
+      - "**.gif"
+      - "**.svg"
+      - "adr/**"
+      - "docs/**"
+
+# Abort prior jobs in the same workflow / PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  validate:
+    runs-on: windows-latest
+    steps:
+      - name: "Dependency: Install Golang"
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
+
+      - name: "Dependency: Install Scoop+Make+NodeJS"
+        shell: pwsh
+        run: |
+          Set-ExecutionPolicy RemoteSigned -scope CurrentUser
+          iex "& {$(irm get.scoop.sh)} -RunAsAdmin"
+          Join-Path (Resolve-Path ~).Path "scoop\shims" >> $Env:GITHUB_PATH
+          scoop install make@4.3 nodejs@16.17.1
```

Comment:
Was there a reason we needed to install these again? https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md

Reply:
I didn't know that it had all the deps we needed!  Was just working off assumption is was a clean Windows box and we had to handle all deps.